js/src/server/privileges.js

```diff
@@ -31,6 +31,29 @@ function checkAddUser (theForm) {
     return Functions.checkPassword($(theForm));
 }
 
+/**
+ * Export privileges modal handler
+ *
+ * @param {object} data
+ *
+ * @param {object} modal
+ *
+ * @param {JQuery} msgbox
+ *
+ */
+function exportPrivilegesModalHandler (data, modal, msgbox) {
+    if (typeof data !== 'undefined' && data.success === true) {
+        modal.find('.modal-body').first().html(data.message);
+        $('#exportPrivilegesModalLabel').first().html(data.title);
+        Functions.ajaxRemoveMessage(msgbox);
+        // Attach syntax highlighted editor to export dialog
+        Functions.getSqlEditor(modal.find('textarea'));
+    } else {
+        modal.modal('hide');
+        Functions.ajaxShowMessage(data.error, false);
+    }
```

Comment:
```suggestion
    if (typeof data !== 'undefined' && data.success === true) {
        modal.find('.modal-body').first().html(data.message);
        $('#exportPrivilegesModalLabel').first().html(data.title);
        Functions.ajaxRemoveMessage(msgbox);
        // Attach syntax highlighted editor to export dialog
        Functions.getSqlEditor(modal.find('textarea'));
        return;
    }
    modal.modal('hide');
    Functions.ajaxShowMessage(data.error, false);
```

That will avoid the useless else